docs/modules/user-guide/partials/proc_adding-a-command-group-to-a-devfile.adoc

```diff
@@ -0,0 +1,56 @@
+[id="proc_adding-a-command-group-to-a-devfile_{context}"]
+= Adding a command group to a devfile
+[role="_abstract"]
+
+Create command groups to help the automation process of your devfile.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-projects-to-a-devfile.adoc[]
+
+.Procedure
+
+. Assign a given command to one or more groups that represent the nature of the command.
```

Comment:
We need to improve this section.

Reply:
Please see the current changes I've made. Happy to expand the abstracts and polish the procedure steps now. Unless we think it can wait for a follow up issues, and I can make those same changes to the procedures throughout.